addons/mail/static/src/components/discuss/discuss.xml

```diff
@@ -3,19 +3,20 @@
 
     <t t-name="mail.Discuss" owl="1">
         <t t-if="discussView">
-            <div class="o_Discuss" t-attf-class="{{ className }}"
+            <div class="o_Discuss d-flex h-100" t-attf-class="{{ className }}"
                 t-att-class="{
                     'o-adding-item': discussView.discuss.isAddingChannel or discussView.discuss.isAddingChat,
-                    'o-mobile': messaging.device.isMobile,
+                    'o-mobile flex-column bg-white': messaging.device.isMobile,
                 }"
                 t-ref="root"
             >
                 <t t-if="!messaging.device.isMobile">
-                    <DiscussSidebar className="'o_Discuss_sidebar bg-light border-right'" localId="discussView.localId"/>
+                    <DiscussSidebar className="'o_Discuss_sidebar pt-3 flex-shrink-0 overflow-auto bg-light border-right'" localId="discussView.localId"/>
                 </t>
                 <t t-if="messaging.device.isMobile" t-call="mail.Discuss.content"/>
                 <t t-else="">
-                    <div class="o_Discuss_content">
+                    <t t-set="o_Discuss_noThread_bg" t-value="False"/>
+                    <div t-attf-class="o_Discuss_content d-flex flex-column overflow-auto w-100 {{ o_Discuss_noThread_bg === True ? 'bg-white' : '' }}">
```

Comment:
Should be `h-100`, not `w-100`, when looking at literal translation from SCSS to bootstrap classes.

Reply:
Yes, much better use `flex` for `flex`. `height/width` leaks details on `flex-column`/`flex-row`, and scales poorly if we change this and/or want to show this component in different `flex-flow`, for some reasons.

Reply:
Done! 👍